core/schema_utils.py

```diff
@@ -681,3 +681,21 @@ def is_valid_username_string(obj: str) -> bool:
             return True
         except utils.ValidationError:
             return False
+
+    @staticmethod
+    def has_allowed_audio_extension(obj: str) -> bool:
```

Comment:
@vojtechjelinek Not sure on what should be the scope of schem_utils file i.e, whether it should hold information about audio extensions?

How about having `should_end_with(choices, strict=Fale)`?

Reply:
Yeah, good point, lets do it the way that @DubeySandeep suggests.

Reply:
@DubeySandeep I will make the changes as you suggested. I did not understand the use of "match_cases" parameter in the function. Can you elaborate?